### PR TITLE
Add composite queries to public roadmap

### PR DESCRIPTION
**Pull request by ulan**

### CODE REVIEW

File: roadmap/1_core-protocol/in-progress/12_composite_queries.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Composite queries
+links:
+  Forum Link: https://forum.dfinity.org/t/proposal-composite-queries/15979
+  Proposal: https://dashboard.internetcomputer.org/proposal/87599
+eta: 2022
```
Comment (manudrijvers) [inline roadmap/1_core-protocol/in-progress/12_composite_queries.md:6]: Are we sure this is feasible? I want to avoid overpromising

**Review by manudrijvers — APPROVED** [review]
looks great, thanks!

File: roadmap/1_core-protocol/in-progress/12_composite_queries.md (hunk 1 above)
Comment (ulan) [reply roadmap/1_core-protocol/in-progress/12_composite_queries.md]: Good point. I bumped it up to 2023. We can adjust it when we are closer to the release.

Comment (ulan) [reply roadmap/1_core-protocol/in-progress/12_composite_queries.md]: @dprats: could you please take another look?

**Pull request merged**